lib/store/chat.dart

```diff
@@ -460,8 +526,10 @@ class ChatRepository extends DisposableInterface
       if (id == monolog?.chat.value.id) {
         _monologShouldBeHidden = false;
         _chats[id] = await _putEntry(monolog!);
+        _paginationChats[id] = _chats[id]!;
       } else if (chat != null) {
         _chats[id] = chat;
+        _paginationChats[id] = _chats[id]!;
```

Comment:
И здесь можем 
```suggestion
        _chats[id] = chat;
        _paginationChats[id] = chat;
```

Это ведь одна и та же ссылка на один и тот же объект, разве нет? Тут же не происходит переноса ownership'а в `_chats[id]`?